library-best-practices/JLBP-17.md

```diff
@@ -0,0 +1,30 @@
+[JLBP-17] Coordinate Major Version Adoption
+-------------------------------------------
+
+It is clearly undesirable for any particular library dependency tree to contain
+conflicts. Unfortunately, for any consumer of such a dependency tree that tries
+to use the highest versions available of all libraries contained in it, there
+are conflicts as soon as a breaking change is introduced. Thus, breaking changes
+need to be propagated as quickly as possible so that such consumers can continue
+using the highest versions.
+
+Perform the rollout in this manner:
+
+1. Decide whether to break the feature as an atomic change or to perform the
+   change in two passes (deprecate then delete).
```

Comment:
deprecate,

Reply:
I assume comma before delete actually? (That's the change I'll try out)